mlir/lib/Dialect/PDL/IR/Builtins.cpp

```diff
@@ -1,5 +1,19 @@
+#include <cassert>
+#include <cstdint>
+#include <llvm/ADT/APFloat.h>
+#include <llvm/ADT/APInt.h>
+#include <llvm/ADT/APSInt.h>
+#include <llvm/ADT/ArrayRef.h>
+#include <llvm/ADT/TypeSwitch.h>
+#include <llvm/Support/Casting.h>
+#include <llvm/Support/ErrorHandling.h>
 #include <mlir/Dialect/PDL/IR/Builtins.h>
+#include <mlir/IR/Attributes.h>
+#include <mlir/IR/BuiltinAttributes.h>
+#include <mlir/IR/Location.h>
 #include <mlir/IR/PatternMatch.h>
+#include <mlir/Support/LLVM.h>
```

Comment:
I am not seeing this being used extensively in MLIR. Maybe we don't needed because we are actually including other headers?